app-misc/terminal-colors/terminal-colors-3.0.2.ebuild

```
@@ -0,0 +1,26 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..10} )
+PYTHON_REQ_USE="ncurses"
+
+inherit python-r1
+
+DESCRIPTION="A tool to display color charts for 8, 16, 88, and 256 color terminals"
+HOMEPAGE="http://zhar.net/projects/shell/terminal-colors"
+SRC_URI="https://github.com/eikenb/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~x64-macos"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}"
+
+src_install() {
+	mv "${S}/${P}/${PN}" "${S}" || die "moving the script failed"
+	rm -rf "${S}/${P}" || die "rm failed"
+	python_foreach_impl python_newscript ${PN} ${PN}
+}
```

```suggestion
src_install() {
	python_foreach_impl python_newscript ${PN} ${PN}
}
```

Although I personally find the README file useful, so you can add `einstalldocs` call to your `Src_install`.